common/laws/00_governance_principles.txt

```diff
@@ -327,6 +326,8 @@ law_council_republic = {
 		country_machinists_pol_str_mult = 0.1
 		country_farmers_pol_str_mult = 0.1
 		interest_group_ig_trade_unions_pop_attraction_mult = 0.13
+		country_authority_add = 25
+		country_law_enactment_time_mult = 0.10
```

Comment:
Slower enactment time?

Reply:
Will make a # note about the changes for balance purposes